_includes/markdown/CSS.md

```diff
@@ -4,377 +4,618 @@ At 10up, we value content and the experience users will have reading it. We writ
+**Note:** Keep this record where engineers will look first—typically in a /decisions/ directory inside the repository's setup or onboarding documentation. Follow your organization's standard documentation format so the notes remain consistent, searchable, and easy to maintain.
 
-Performance best practices are not only for the browser experience, but for code maintenance as well.
+<h2 id="content-agnostic" class="anchor-heading">Assume You Know Nothing About the Content {% include Util/link_anchor anchor="content-agnostic" %}</h2>
 
-<h2 id="responsive-design" class="anchor-heading">Responsive Design {% include Util/top %}</h2>
+When building components, always start with the mindset that you have no control over the content they’ll handle. Even if the client or designer specifies certain limits, assume those constraints could change—or be ignored entirely.
 
-We build our websites mobile first. We do not rely on JavaScript libraries such as `respond.js` as it does not work well in certain environments. Instead, we leverage a natural, mobile-first build process and allow sites gracefully degrade.
+Content is unpredictable. You might not know how long a title will be, how many items a navigation menu will have, or the length of an excerpt in a card. Design decisions made today can quickly be overturned tomorrow when new requirements come in.
 
-### Min-width media queries
+Prepare for edge cases. Your components must be resilient and robust enough to handle all reasonable variations, including unexpected ones. If you don’t account for these possibilities, it could lead to broken layouts, unreadable text, or poor user experiences.
 
-A responsive website should be built with min-width media queries. This approach means that our media queries are consistent, readable and minimize selector overrides.
+**Useful resources:**
 
-* For most selectors, properties will be added at later breakpoints. This way we can reduce the usage of overrides and resets.
-* It targets the least capable browsers first which is philosophically in line with mobile first — a concept we often embrace for our sites
-* When media queries consistently "point" in the same direction, it makes it easier to understand and maintain stylesheets.
+- [Defensive CSS](https://defensivecss.dev/)
 
-Avoid mixing min-width and max-width media queries.
+### Stress Test Your Components
 
-### Breakpoints
+When building a component, test it against extreme content scenarios. For example:
 
-Working with build tools that utilize Sass or PostCSS processing, we can take advantages of reusability and avoid having an unmaintainable number of breakpoints. Using variables and reusable code blocks we can lighten the CSS load and ease maintainability.
+* A navigation menu with 20 items instead of 5.  
+* A card title that wraps onto three or more lines.  
+* A card title that is just one word.  
+* An excerpt that's 300 characters long—or empty.
```

Comment:
suggestion: "A card without a thumbnail" <-- this I see happening more often than not.